robert_beverage/scripts/web_server

```diff
@@ -1,5 +1,35 @@
 #!/usr/bin/env python
 import rospy
+import socket
+import SocketServer
+import SimpleHTTPServer
+from std_msgs.msg import String
 
 rospy.init_node('web_server_node')
-rospy.loginfo('Inside web_server script')
\ No newline at end of file
+rospy.loginfo('Inside web_server script')
+
+HOST = socket.gethostname()
+PORT = 8080
+address = ("", PORT)
+pub = rospy.Publisher('chatter', String, queue_size=10)
+rate = rospy.Rate(10)
+
+
+class RequestHandler(SimpleHTTPServer.SimpleHTTPRequestHandler):
+    def do_GET(self):
+        if self.path.startswith("/beverage"):
+            info = self.path.strip("/")[9:]
+            coor = dict(info.split("=").split("&"))
+        while not rospy.is_shutdown():
```

Comment:
This loop will prevent browser from returning data to client. It is not needed here.